assignment2.txt

```diff
@@ -0,0 +1,36 @@
+1. Can two or more operators be combined in a single line of program code?
+Yes, although it must be done syntactically correct. Multiple operators such as *,%,/,+, and -
+can all be used in a single line between numbers. Ex: int g = 5 % 2 + 1; Order of operations will
+still apply. Other operators such as >, <, ==, and != can also be used multiple times in
+a single line of code, however they must be used across multiple expressions. Using the example
+from question 4, you cannot use the conditional statement 0 < x < 15. In order to get the
+logical equivalent that is syntactically correct, you must separate it into two expressions
+with a logical operator in between: 0 < x && x < 15.
+2. What are logical errors and how are they different from syntax errors?
```

Comment:
right - it's important to note that logical errors occur during runtime whereas syntax errors occur at compile time.